src/app/component/header/header.component.html

```diff
@@ -1,10 +1,10 @@
 <nav class="navbar fixed-top navbar-expand-sm navbar-light bg-light" style="padding: 0px;">
   <div class="container-fluid">
     <a href="#" class="navbar-brand h1" routerLink="/dashboard" style="margin: 0px;">
-      <img class="d-inline-block" src="assets/img/logo1.png" width="35" height="35" />
+      <img class="d-inline-block" src="assets/img/logo1.png" alt="Jyoti Techn" width="35" height="35" />
       <span>APPLICATION</span></a>
         
-      <div class="navigation">
+      <div class="navigation" *ngIf="userName">
```

Comment:
could you please change name userName to isUserLoggedIn ?

Reply:
For using this I need to create a new variable isUserLoggedIn  for single if which is waste of memory